Source/JavaScriptCore/API/JSScript.mm

```diff
@@ -24,12 +24,14 @@
  */
 
 #import "config.h"
+#import "JSBaseInternal.h"
```

Comment:
This is a code style violation (files start with config.h and their own include), and thus suspicious. Also, we don't include JSBaseInternal.h essentially anywhere else, and JSBase.h includes are in headers.

But I'm not familiar with this code, so someone who works on JSC needs to weigh in.

Reply:
This is needed to bring in the `JSC_OBJC_API_ENABLED` define, let me know if there's a better way.

Reply:
(I changed it to `#import "JSBase.h"` which is the one defining `JSC_OBJC_API_ENABLED`)

`JSScriptInternal.h` has this as its first line:
```
#if JSC_OBJC_API_ENABLED
```
so the macro `JSC_OBJC_API_ENABLED` needs to be defined before including it.